src/Calendar/Calendar.js

```diff
@@ -705,14 +715,10 @@ Calendar.basePropTypes = {
         /** aria-label for previous button when years are displayed */
         show12PreviousYears: PropTypes.string
     }),
-    /** Object with special dates and special date types in shape of `{\'YYYYMMDD\': type}`. Type must be a number between 1-20 */
-    specialDays: PropTypes.object
-};
-
-Calendar.propTypes = {
-    ...Calendar.basePropTypes,
```

Comment:
Seems like we could do the same thing for `Time.basePropTypes`?

Reply:
Ooo good call! Missed that in the first iteration